site-src/v1alpha2/guides/http-redirect-rewrite.md

```diff
@@ -0,0 +1,88 @@
+# HTTP path redirects and rewrites
+
+[HTTPRoute resources](/v1alpha2/api-types/httproute) can issue redirects to
+clients or rewrite paths sent upstream using
+[filters](/v1alpha2/api-types/httproute#filters-optional). This guide shows how
+to use these features.
+
+Note that redirect and rewrite filters are mutually incompatible. Rules cannot
+use both filter types at once.
+
+## Redirects
+
+Redirects return HTTP 3XX responses to a client, instructing it to retrive a
+different resource. [`RequestRedirect` rule
+filters](v1alpha2/references/spec/#gateway.networking.k8s.io/v1alpha2.HTTPRequestRedirectFilter)
+instruct Gateways to emit a redirect response to requests matching a filtered
+HTTPRoute rule.
+
+Redirect filters can substitute various URL components independently. For
+example, to issue a permanent redirect (301) from HTTP to HTTPS, configure
+`requestRedirect.statusCode=301` and `requestRedirect.scheme="https"`:
+
+```yaml
+{% include 'experimental/http-redirect-rewrite/httproute-redirect-https.yaml' %}
+```
+
+Redirects change configured URL components to match the redirect configuration
+while preserving other components from the original request URL. In this
+example, the request `GET http://redirect.example/cinammon` will result in a
+301 response with a `location: https://redirect.example/cinnamon` header. The
+hostname (`redirect.example`), path (`/cinnamon`), and port (implicit) remain
+unchanged.
+
+Redirect filters do not return redirects if the redirected URL and original URL
+are the same. Although the above configuration applies to both HTTP and HTTPS
+requests, HTTPS requests will proxy upstream, not return a redirect response
+that would result in a loop.
+
+### Path redirects
```

Comment:
I think this section needs an "Experimental Channel" disclaimer like what I have below:
```
!!! info "Experimental Channel"

    The `Port` field described below is currently only included in the
    "Experimental" channel of Gateway API. For more information on release
    channels, refer to the [related documentation](https://gateway-api.sigs.k8s.io/concepts/versioning).
```